archives/archives_test.go

```diff
@@ -491,7 +498,14 @@ func TestArchiveOrgRuns(t *testing.T) {
 }
 
 func TestArchiveActiveOrgs(t *testing.T) {
-	_, rt := setup(t)
+	ctx, rt := setup(t)
+
+	err := rt.CW.Send(ctx,
+		types.MetricDatum{MetricName: aws.String("NumGoats"), Value: aws.Float64(10), Unit: types.StandardUnitCount},
+		types.MetricDatum{MetricName: aws.String("NumSheep"), Dimensions: []types.Dimension{{Name: aws.String("Host"), Value: aws.String("foo1")}}, Value: aws.Float64(20), Unit: types.StandardUnitCount},
```

Comment:
Still don't feel this belongs here.. it's like testing the DB connection. Running archiving will test that `rt.CW.Send` works as much as these lines.